rfc/updates.md

```diff
@@ -267,3 +267,28 @@ Upgrading Constellation microservices to 2.2.0:
 Warning: Please backup any important components before upgrading Kubernetes
 Apply change [yes/No]?
 ```
+
+# Compatibility
+`constellation-upgrade` has to handle the version of four components: CLI, image, microservices and Kubernetes.
+To do this correctly and keep the cluster in a working condition some constraints are required.
+
+- Microservice, image and CLI are versioned in lockstep, i.e. each release a new version of all componentes is released.
+  Each of these version numbers is one Constellation version.
+- Each Constellation version is compatible to three or four Kubernetes versions.
+  When a new Kubernetes version is released Constellation will support four versions for one release cycle; before phasing out the oldest Kubernetes version.
+  To learn if microserviceVersion A.B.C is compatible with Kubernetes version X.Y, one has to check whether the Constellation version is compatible with the Kubernetes version X.Y.
+  The Constellation version will be the same as the microserviceVersion: A.B.C. Same goes for the image and CLI.
```

Comment:
We've already established this a few points before.

Reply:
Can't find where we explicitly state it like that. Doesn't hurt to repeat it?

Reply:
I think we already somewhat mention this in the first point:
> Constellation microservices, VM image and the CLI are versioned in lockstep, i.e. each release a new version of all components is released.
  The Constellation version references all of these components.

If we want to also mention the semver, I think we should do this there. If needed we can then say something like `since the microserviceVersion is the same as the CLI version ...`.